pom.xml

```diff
@@ -5982,6 +5988,11 @@
           <artifactId>exec-maven-plugin</artifactId>
           <version>1.5.0</version>
         </plugin>
+        <plugin>
+          <groupId>org.apache.avro</groupId>
+          <artifactId>avro-maven-plugin</artifactId>
```

Comment:
What's the plugin used for?

Reply:
This plugin can be used to generate avro object from a schema it was for testing, removed.